src/lib/getFileMimetype.d.ts

```
@@ -0,0 +1,6 @@
+import type { FileType } from './fileTypes'
+
+export function getFileMimetype(
+  mime?: string,
+  name?: string
+): FileType | null
```

ESLint

[error] Replace `⏎··mime?:·string,⏎··name?:·string⏎` with `mime?:·string,·name?:·string`

(prettier/prettier)